client/src/client/components/Dashboard/Dashboard.jsx

```diff
@@ -1,92 +1,114 @@
-import React, { useState, useEffect } from 'react';
-import Navbar from '../Navbar/Navbar.jsx';
-import Featured from '../FeaturedCard/Featured.jsx';
-import SessionCard from '../SessionCard/SessionCard.jsx';
-import Filter from '../Filter/Filter.jsx';
-import './Dashboard.css';
-import "../styles.module.css"
-import Divider from '@mui/material/Divider';
-import Fab from '@mui/material/Fab';
-import AddIcon from '@mui/icons-material/Add';
-import CreateSessionPopup from '../CreateSession/CreateSessionPopup.jsx';
-import SessionMoreInfoPopup from '../SessionMoreInfo/SessionMoreInfoPopup.jsx';
+import React, { useState, useEffect } from "react";
+import Navbar from "../Navbar/Navbar.jsx";
+import Featured from "../FeaturedCard/Featured.jsx";
+import SessionCard from "../SessionCard/SessionCard.jsx";
+import Filter from "../Filter/Filter.jsx";
+import "./Dashboard.css";
+import "../styles.module.css";
+import Divider from "@mui/material/Divider";
+import Fab from "@mui/material/Fab";
+import AddIcon from "@mui/icons-material/Add";
+import CreateSessionPopup from "../CreateSession/CreateSessionPopup.jsx";
+import SessionMoreInfoPopup from "../SessionMoreInfo/SessionMoreInfoPopup.jsx";
 import { useSelector } from "react-redux";
-import { useDispatch } from 'react-redux';
-
+import { useDispatch } from "react-redux";
 
 export default function Dashboard() {
-    const dispatch = useDispatch();
-
-    const [isCreateSessionModalOpen, setIsCreateSessionModalOpen] = useState(false);
-    const [isSessionMoreInfoModalOpen, setIsSessionMoreInfoModalOpen] = useState(false);
-    const [selectedSessionId, setSelectedItemId] = useState(null);
-
-    const sessions = useSelector(store => store.sessionReducer).sessions;
+  const dispatch = useDispatch();
 
+  const [isCreateSessionModalOpen, setIsCreateSessionModalOpen] =
+    useState(false);
+  const [isSessionMoreInfoModalOpen, setIsSessionMoreInfoModalOpen] =
+    useState(false);
```

Comment:
This on purpose or is it from a linter? Just wondering why we're putting the useState on a new line?

Reply:
`Prettier`'s code formatting. I think the line was too long